local/Dockerfile

```diff
@@ -1,24 +1,27 @@
 FROM python:3.10-alpine@sha256:ef5536e8f22c1697807cabe68765567cea4558b4b3f3287e2277b336cf2273a5
 
 ENV PYTHONDONTWRITEBYTECODE 1
+ENV POETRY_VERSION "1.3.2"
 
 RUN apk add --no-cache bash build-base git gcc musl-dev postgresql-dev g++ make libffi-dev libmagic libcurl curl-dev && rm -rf /var/cache/apk/*
 
 # update pip
-RUN python -m pip install wheel
+RUN python -m pip install wheel poetry==${POETRY_VERSION}
 
 RUN set -ex && mkdir /app
 
 WORKDIR /app
 
-COPY requirements.txt /app
-RUN set -ex && pip3 install -r requirements.txt
+# COPY requirements.txt /app
+# RUN set -ex && pip3 install -r requirements.txt
 
 COPY . /app
 
+RUN poetry install
+
 ENV PORT=6011
 
 ARG GIT_SHA
 ENV GIT_SHA ${GIT_SHA}
 
-CMD ["sh", "-c", "gunicorn -c gunicorn_config.py application"]
\ No newline at end of file
+CMD ["poetry", "run", "gunicorn -c gunicorn_config.py application"]
```

Comment:
need the "sh", "-c" here probably? Mind you, I don't think this file is used for anything.

Reply:
Correct, good catch